docroot/profiles/custom/cgov_site/modules/custom/cgov_core/config/install/workflows.workflow.editorial_workflow.yml

```diff
@@ -0,0 +1,127 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - content_moderation
+id: editorial_workflow
+label: 'Editorial Workflow'
+type: content_moderation
+type_settings:
+  states:
+    archive_requested:
+      published: false
+      default_revision: false
+      label: 'Archive Requested'
+      weight: 1
+    archived:
+      published: false
+      default_revision: true
+      label: Archived
+      weight: 2
+    draft:
+      label: Draft
+      published: false
+      default_revision: false
+      weight: -4
+    editing:
+      published: false
+      default_revision: false
+      label: Editing
+      weight: -1
+    post_publication_review:
+      published: false
+      default_revision: false
+      label: 'Post-Publication Review'
+      weight: 0
+    published:
+      label: Published
+      published: true
+      default_revision: true
+      weight: -2
+    review:
+      published: false
+      default_revision: false
+      label: Review
+      weight: -3
+  transitions:
+    approve:
+      label: 'Approve Edits of Published Content'
+      from:
+        - post_publication_review
+      to: published
+      weight: 5
+    approve_archive:
+      label: 'Approve Archive Request'
+      from:
+        - archive_requested
+      to: archived
+      weight: 8
+    back_to_draft:
+      label: 'Back To Draft'
+      from:
+        - review
+      to: draft
+      weight: -1
+    back_to_editing:
+      label: 'Back To Editing'
+      from:
+        - post_publication_review
+      to: editing
+      weight: 4
+    publish:
+      label: Publish
+      from:
+        - review
+      to: published
+      weight: 0
+    quick_publish:
+      label: 'Quick Publish'
+      from:
+        - draft
+      to: published
+      weight: 1
+    quick_publish_2:
+      label: 'Quick Publish (Replace Existing Published Revision)'
+      from:
+        - editing
+      to: published
+      weight: 6
+    reject_archive:
+      label: 'Reject Archive Request'
+      from:
+        - archive_requested
+      to: published
+      weight: 9
+    request_archive:
+      label: 'Request Archive'
+      from:
+        - published
+      to: archive_requested
+      weight: 7
+    request_review:
+      label: 'Request Review'
+      from:
+        - draft
+      to: review
+      weight: -2
+    request_review_2:
+      label: 'Request Review of Published Content'
+      from:
+        - editing
+      to: post_publication_review
+      weight: 3
+    resurrect_archived_content:
+      label: 'Resurrect Archived Content'
```

Comment:
Maybe `republish_archived_content` instead of `resurrect`

Reply:
No roles appear to have the `resurrect_archived_content` permission

Reply:
> Maybe `republish_archived_content` instead of `resurrect`

Except it's not being republished in this transition. The author is just dragging the content back to the `draft` state. Republication may (or may not) happen later.

Reply:
Perhaps `remove_from_mothballs`? 😃